mobile/lib/shared/ui/scaffold_error_body.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:immich_mobile/extensions/build_context_extensions.dart';
+
+// Error widget to be used in Scaffold when an AsyncError is received
+class ScaffoldErrorBody extends StatelessWidget {
+  final IconData icon;
+
+  const ScaffoldErrorBody({this.icon = Icons.ac_unit, super.key});
```

Comment:
why is the default `ac_unit`?

Reply:
I don't really have a reason for that. The Ice Crystal icon looked good with the Error Occurred text above. I'm open for suggestions to change it though 😅

Reply:
`error_outline` as it is the generic error icon?